app/favourites.py

```diff
@@ -16,6 +16,23 @@
 favourites = Blueprint("favourites", __name__, template_folder='templates')
 
 
+@favourites.route("/", methods=["GET", "POST"])
+@login_required
+def view_favourites():
+    """
+    Function to display user's favourite companies
+    - require user to be logged in
+    - retrieve user in session
+    - find organisations in list of user's favourites
+    """
+    user = User.find_one_user(session["_user_id"].lower())
+    favourites = list(mongo.db.organisations.find({"_id": {
+                                                    "$in": user["favourites"]}
```

Comment:
I am getting a `KeyError` here because favourites is not set in the `app/auth.py`'s register view. You can add it after line 32.